rllib/core/models/torch/encoder.py

```diff
@@ -26,23 +31,25 @@
 
 
 class TorchMLPEncoder(TorchModel, Encoder):
-    def __init__(self, config: ModelConfig) -> None:
+    def __init__(self, config: MLPEncoderConfig) -> None:
         TorchModel.__init__(self, config)
         Encoder.__init__(self, config)
 
         # Create the neural networks
         self.net = TorchMLP(
             input_dim=config.input_dims[0],
             hidden_layer_dims=config.hidden_layer_dims,
-            output_dim=config.output_dims[0],
             hidden_layer_activation=config.hidden_layer_activation,
+            hidden_layer_use_layernorm=config.hidden_layer_use_layernorm,
+            output_dim=config.output_dims[0],
+            use_bias=config.use_bias,
         )
 
     @override(Model)
     def get_input_spec(self) -> Union[Spec, None]:
         return SpecDict(
             {
-                SampleBatch.OBS: TorchTensorSpec("b, h", h=self.config.input_dims[0]),
+                SampleBatch.OBS: TorchTensorSpec("b, d", d=self.config.input_dims[0]),
```

Comment:
I renamed these "hanging right-side dims" to `d`. I feel like `h` should be used only for image height and LSTM internal h-state.

Reply:
d=dims